clean-leather.lic

```diff
@@ -26,23 +27,36 @@ class CleanLeather
 
     ensure_copper_on_hand(2000, @settings)
 
-    @preservative = args.noun == 'bone' ? 'bleaching solution' : 'tanning lotion'
+    @preservative = args.noun =~ /^bones?$/i ? 'bleaching solution' : 'tanning lotion'
```

Comment:
Uncleaned bones are usually "bones" not "bone", and using the actual noun caused the script to fail.

Reply:
❓ "usually"... should the 's' be optional in the regex?